tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -152,23 +163,27 @@ sub run_test {
         $timeout = 300;
     }
 
-    my $test_cmd      = $self->get_script_run();
-    my $script_output = $self->execute_script_run($test_cmd, $timeout);
-
-    if ($add_junit_log_flag eq "yes") {
-        $self->add_junit_log($script_output);
+    my $test_cmd = $self->get_script_run();
+    if (check_var('ARCH', 's390x')) {
+       $self->svirt_cmd ("$test_cmd");
```

Comment:
If you want to design s390x  as not having all other options like $timeout, $assert_pattern, $add_junit_log_flag ..., please add comment at the beginning of this function to describe it.